department-of-veterans-affairs/caseflow
# Account for TZ in EvidenceSubmissionWindow End Date during Substitution

Connects [CASEFLOW-1568](https://vajira.max.gov/browse/CASEFLOW-1568)

### Description
This updates the logic for calculating the end date of a new `EvidenceSubmissionWindowTask` created during appellant substitution to be set at midnight of the user's time zone, not UTC. Previously it was saving as midnight UTC, so to users in the US, it appeared as being one day earlier than anticipated.

### Acceptance Criteria
- [ ] New `EvidenceSubmissionWindowTask.timer_ends_at` is midnight w/ user's TZ offset

### Testing Plan
1. Log in as `COB_USER`
2. Perform appellant substitution for Appeal with `evidence_submission` docket type
3. Be sure to select the `EvidenceSubmissionWindowTask` during substitution
4. Ensure that `timer_ends_at` on the new `EvidenceSubmissionWindowTask` returns a value with appropriate TZ offset (not UTC)
